src/lm_buddy/integrations/huggingface/asset_loader.py

```diff
@@ -66,23 +63,18 @@ class HuggingFaceAssetLoader:
     def __init__(self, artifact_loader: ArtifactLoader):
         self._artifact_loader = artifact_loader
 
-    def resolve_asset_path(self, path: HuggingFaceAssetPath) -> tuple[str, str | None]:
-        """Resolve the actual HuggingFace name/path from a config.
-
-        Currently, two config types contain references to a loadable HuggingFace path:
-        (1) A `HuggingFaceRepoConfig` that contains the repo path directly
-        (2) A `WandbArtifactConfig` where the filesystem path is resolved from the artifact
-        """
+    def resolve_asset_path(self, path: LoadableAssetPath) -> str:
```

Comment:
Note that I've changed the signature of this method to only return a string path, rather than a string path AND a `revision` field.

I felt that included `revision` within the path resolution logic was a poor abstraction for the following reasons:
- It is only applicable to HF Hub repos, not artifacts or direct filesystem paths
- We aren't even really using it

If we want to add back the ability to load from a specific HF Hub repo revision, we can include that parameter as part of a separate config (e.g., on the `AutoModelConfig` directly).